lib/utils/docker.coffee

```diff
@@ -156,15 +162,33 @@ exports.runBuild = (params, options, getBundleInfo, logStreams) ->
 	resolver = require('resin-bundle-resolve')
 	es = require('event-stream')
 	doodles = require('resin-doodles')
+	transpose = require('docker-qemu-transpose')
+	path = require('path')
 
 	logging = require('../utils/logging')
 
 	# The default build context is the current directory
 	params.source ?= '.'
 	logs = ''
+	# Only used in emulated builds
+	qemuPath = ''
 
-	# Tar up the directory, ready for the build stream
-	tarDirectory(params.source)
+	Promise.try ->
+		return if not (options.emulated and platformNeedsQemu())
+
+		hasQemu()
+		.then (present) ->
+			if !present
+				logging.logInfo(logStreams, 'Installing qemu for ARM emulation...')
+				installQemu()
+		.then ->
+			# Copy the qemu binary into the build context
+			copyQemu(params.source)
+		.then (binPath) ->
+			qemuPath = binPath.split(path.sep)[1...].join(path.sep)
```

Comment:
The path of the binary before this is:
```
[context location]/.resin/qemu
```
but inside of the build, this should be
```
.resin/qemu
```